### DISCUSSION TITLE
Bounding box ValueError: has an area of zero.

### DISCUSSION THREAD

**Discussion by ghost** [jsvine/pdfplumber#394] [Ask for help with specific PDFs]
Hi,

I try to crop my pages, but I always receive the following ValueError
"Bounding box ... has an area of zero."

This is what I used:
```python
filename = "test.pdf"
box = (0.9*float(page.width), page.height, page.width, page.height)

with pdfplumber.open(filename, password = "") as pdf:
    page = pdf.pages[0]
    cropped_page = page.crop(box, relative=False) 
    im = cropped_page.to_image()
    image = im.draw_rects(cropped_page.extract_words())
    #im.save("image.png", format="PNG")
image
```

Solved it with the following ajustment of the box, but are not able to delete the post. Sorry
```python
box = (0.1*float(page.width), 0, page.width, page.height)
```

**Comment by jsvine** [comment]
> Solved it with the following ajustment of the box, but are not able to delete the post.

Yep, looks like you answered/solved your question yourself!
